### PR TITLE
VACMS-18512: Update va-link for event details page to remove duplicate events

### PR DESCRIPTION
**Pull request by chriskim2311**
## Summary
This PR updates the events detail page to remove duplicative ga events and change the directions for google maps links.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18512

## Testing done
Tested locally `/durham-health-care/events/69410/`

## Screenshots

VA_Greater_Los_Angeles_Health_Care___Veteran_Produce_Distribution_Events___Veterans_Affairs

VA_Greater_Los_Angeles_Health_Care___Veteran_Produce_Distribution_Events___Veterans_Affairs

VA_Durham_Health_Care___Greenville_HCC_PACT_Act_And_PRIDE_Event___Veterans_Affairs

## What areas of the site does it impact?
Event Detail Page

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

**Review by randimays — COMMENTED** [review]
Code looks good, but could you provide screenshots with Adswerve demonstrating the remaining event(s)?

**Comment by chriskim2311** [comment]
@randimays Added the adswerve screenshots!

File: src/site/includes/directions-google-maps.liquid [hunk 1]
```diff
@@ -6,18 +6,11 @@ directionsLinkOnClickPropName
...
-
 <div>
-  <a
-    {{ onClick | strip }}
-    href="https://maps.google.com?saddr=Current+Location&amp;daddr={{ directionsLinkAddress | strip }}">
-    Get directions on Google Maps
-    <span class="sr-only">to {{ directionsLinkTitle | strip }}</span>
-  </a>
+  <va-link
+    aria-label="to {{ directionsLinkTitle | strip }}"
```
Comment (randimays) [inline src/site/includes/directions-google-maps.liquid:11]: Sorry I didn't catch this before; I think this has to be `label` in order for the `aria-label` to apply to the `<a>` tag inside.

Comment (chriskim2311) [reply src/site/includes/directions-google-maps.liquid]: Ahh thanks changed!

**Comment by eselkin** [comment]
I know @laflannery had told us not to move Google directions links to va-links on Facilities stuff because they couldn't handle the cohesive screen-reader labeling (Like "get directions on google maps to x" in one phrase read out). Does this work correctly now?

**Comment by laflannery** [comment]
LGTM!

**Review by randimays — APPROVED** [review]
LGTM!

**Pull request merged**